server/routes/shared/appointment/feedback/submittedFeedbackPresenter.test.ts

```diff
@@ -0,0 +1,94 @@
+import actionPlanAppointmentFactory from '../../../../../testutils/factories/actionPlanAppointment'
+import appointmentFactory from '../../../../../testutils/factories/appointment'
+import deliusServiceUserFactory from '../../../../../testutils/factories/deliusServiceUser'
+import User from '../../../../models/hmppsAuth/user'
+import SubmittedFeedbackPresenter from './submittedFeedbackPresenter'
+import { ActionPlanAppointment } from '../../../../models/actionPlan'
+import Appointment from '../../../../models/appointment'
+
+describe(SubmittedFeedbackPresenter, () => {
+  describe('text', () => {
+    function assertText(appointment: ActionPlanAppointment | Appointment): void {
+      const serviceUser = deliusServiceUserFactory.build()
+      const presenter = new SubmittedFeedbackPresenter(appointment, serviceUser)
+      expect(presenter.text).toMatchObject({
+        title: 'View feedback',
+      })
+    }
+    it('includes the title of the page', () => {
+      assertText(actionPlanAppointmentFactory.build())
+      assertText(appointmentFactory.build())
+    })
+  })
+
+  describe('sessionDetailsSummary', () => {
+    describe('when a caseworker is passed in', () => {
+      function assertSessionDetailsWithCaseWorker(appointment: ActionPlanAppointment | Appointment): void {
+        const caseworker: User = {
+          username: 'Kay.Swerker',
+          authSource: 'Delius',
+          userId: '91229a16-b5f4-4784-942e-a484a97ac865',
+        }
+        const serviceUser = deliusServiceUserFactory.build()
+        const presenter = new SubmittedFeedbackPresenter(appointment, serviceUser, caseworker)
+        expect(presenter.sessionDetailsSummary).toEqual([
+          {
+            key: 'Caseworker',
+            lines: ['Kay.Swerker'],
+          },
+          {
+            key: 'Date',
+            lines: ['01 Feb 2021'],
+          },
+          {
+            key: 'Time',
+            lines: ['13:00'],
+          },
+        ])
+      }
+      it('extracts the date and time from the appointmentTime and puts it in a SummaryList format alongside the caseworker name', () => {
+        assertSessionDetailsWithCaseWorker(
```

Comment:
I'd personally find repetition, rather than the re-usable method, clearer to understand here and the test below in terms of tests serving as documentation (particularly with the `it` description) as I have to scroll up to see what the expected output is.